mne/channels/tests/test_interpolation.py

```diff
@@ -12,7 +12,6 @@
 base_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests', 'data')
 raw_fname = op.join(base_dir, 'test_raw.fif')
 event_name = op.join(base_dir, 'test-eve.fif')
-evoked_nf_name = op.join(base_dir, 'test-nf-ave.fif')
```

Comment:
No filter maybe? You could git grep for where it's still used after this PR
and it would indicate it.